utils/utils.h

```diff
@@ -623,6 +627,10 @@ int gbStrdup(char **dest, const char *src,
 char* gbStrcpy(char *dest, const char *src, size_t destbytes,
                const char *filename, const char *funcname, size_t linenr);
 
+char *
+gbStrcat(char *dest, const char *src, size_t destbytes,
```

Comment:
Will it be possible to align this like other declarations in the file ?

Char *  func();

Reply:
Sure, will fix it.